frontend/www/js/omegaup/arena/contest_listv2.ts

```diff
@@ -26,6 +28,57 @@ OmegaUp.on('ready', () => {
           query: this.query,
           tab,
         },
+        on: {
+          'get-next-chunk': (page: number, pageSize: number, query: string) => {
```

Comment:
nit:

```suggestion
          'get-chunk': (page: number, pageSize: number, query: string) => {
```

esto puede regresar el siguiente o el anterior.